ui/app/AppLayouts/Onboarding2/OnboardingLayout.qml

```diff
@@ -208,8 +173,40 @@ Page {
         onFinished: (flow) => d.finishFlow(flow)
     }
 
+    // needs to be on top of the stack
```

Comment:
This is no longer true since you moved it below; why?

Reply:
earlier it was also below the stack view originally, but now the flow is itself StackView, so I had to move even lower

Reply:
OK, I have a gut feeling this will break the mouse back button feature :) Will test anyway when it's no longer WIP

Reply:
basically the order is preserved as it was, I haven't noticed anything suspicious in behavior